src/packages/carpool-view-flow/react/components/common/Time.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import TimeIcon from 'material-ui/lib/svg-icons/action/schedule'
+
+import { config } from '../../config.js';
+
+const commonStyle = {
+  display: 'flex',
+  flexDirection:' row',
+  alignItems: 'center',
+};
+
+const approximateTimeStyle = {
+  ...commonStyle,
+  color: '#888',
+}
+
+const timeStyle = {
+  ...commonStyle,
+}
+
+const Time = (props) => (
+  <div style={props.approximate ? approximateTimeStyle : timeStyle}>
```

Comment:
Two same if's - both they check props.aproximate
In general is Time component contract well defined (having arguments 
  approximate: React.PropTypes.bool,
  time: React.PropTypes.any.isRequired,
)?